src/ParseLiveQuery.js

```diff
@@ -69,16 +70,19 @@ LiveQuery.on('error', () => {
 export default LiveQuery;
 
 let getSessionToken = () => {
-  let currentUser = CoreManager.getUserController().currentUser();
-  let sessionToken;
-  if (currentUser) {
-    sessionToken = currentUser.getSessionToken();
-  }
-  return sessionToken;
+  let promiseUser = CoreManager.getUserController().currentUserAsync();
```

Comment:
what is the issue here? I thought it's more readable than chaining directly from the above.